README.md

```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
According to the Microverse Readme template, your repo readme is lacking this section:

![image](https://user-images.githubusercontent.com/57421823/160064155-d7a03b51-5f27-402e-915e-ddad0ac7d3d7.png)

Therefore, as requested by the previous reviewer:

- [ ] Kindly provide the "Getting started" section on your readme with instructions about how to clone, install and run your project locally.

## Explanation

On the Microverse Readme Template, nothing is optional except for the "Tests" and "Live Demo" sections if not required from the project.

## Hints and further advice

You can create a great readme that meets all requirements once, and then use it for all your projects, just customizing it each time!

That way, we will meet this requirement and will be ready for approval 💪

Reply:
Sorry about that, I totally forgot that section. Thank you so much for your review @Meltrust